_posts/2022-08-14-data-pipelines-production-ready-great-expectations.md

```diff
@@ -0,0 +1,102 @@
+* [Great Expectations meetup to discuss Trino](https://www.youtube.com/watch?v=4SieRmibb0U).
+* [Superconductive](https://superconductive.ai/) joined this year’s mini Trino 
+Summit event 
+[Cinco de Trino](https://www.youtube.com/watch?v=kfJ63DNbAuI&list=PLFnr63che7wYDHjUsmp43THLmAlqPDHlM)
+to showcase using 
+[managed solutions for Great Expectations and Trino](https://www.youtube.com/watch?v=9HE6LawCHP8&list=PLFnr63che7wYDHjUsmp43THLmAlqPDHlM&index=7).
+
+Today, we’re showcasing a demo that showcases a scenario with Trino running as
+the datalake query engine with multiple phases of data transformations on some
+Pokemon data sets. At each phase, we need to validate the data is in the correct
+schema, counts, and various other factors to validate.
+
+So in the context of this demo, we’re going to be using Trino as the query
+engine of a data lakehouse. For this, you can use various table formats. To
+index and read the data, we’re going to use the Hive table format for landing
+tables and Iceberg table format for the structure and consume tables. This is
+mainly because Trino currently doesn’t support reading CSV files from Iceberg 
+yet, but also conveys the level of flexibility you have in a data lake.
+
+## Trino and Great Expectations demo
+
+In this scenario, we’re going to ingest Pokemon pokedex data and Pokemon Go 
+spawn location data which lands as raw CSV files in our data lake. We then use
+Trino’s Hive catalog to read the data from the landing files and clean up and 
+optimize that raw data into more performant ORC files in the structure tables. 
+
+![](/assets/blog/data-pipelines-production-ready-great-expectations/trino-ge-lakehouse.svg)
+
+The last step is to join and transform the spawn data and pokedex data into a
+single table that is cleaned and ready to be utilized by a data analyst, data
+scientist, or other data consumer. Every area of the pipeline where the data is
+transformed opens up a vulnerability as the state can go from good to bed as
+infrastructure fails or is updated as newer versions of the pipeline roll out.
+This is where adding Great Expectations is crucial.
+
+Now that you have a better understanding of the scenario, feel free to watch the
+video below and try running it yourself!
+
+<iframe src="https://www.youtube.com/embed/h6UYOilESfQ" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; 
+margin-bottom:5px; max-width: 100%;" allowfullscreen=""> 
+</iframe>
+
+<a class="btn btn-pink btn-md waves-effect waves-light" href="https://github.com/bitsondatadev/trino-datalake/blob/main/tutorials/expecting-greatness-from-trino.md">Try this Trino demo yourself >></a>
+
+## Conclusion
+
+Data quality is growing in importance as the complexity of data lakes increase.
+It is a necessity that avoids faulty data and improves the trust that data
```

Comment:
"Data quality is a necessity that avoids faulty data" feels a little like a truism. It might be better to highlight why faulty data is a problem (errors, incorrect insights, etc.) and stress that avoiding it is crucial.

Reply:
I just removed the "avoids faulty data" and kept the focus on improving the trust that data consumers have